proto/evmos/epochs/v1/genesis.proto

```diff
@@ -8,7 +8,9 @@ import "google/protobuf/timestamp.proto";
 option go_package = "github.com/tharsis/evmos/v4/x/epochs/types";
 
 message EpochInfo {
-  string identifier = 1;
+  // DEPRECATED: Identifier has been removed;
+  reserved 1;
+  reserved "identifier";
```

Comment:
When I start the chain with `./init.sh` I get multiple comments like:
```
proto: duplicate proto type registered: evmos.epochs.v1.EpochInfo
proto: duplicate proto type registered: evmos.epochs.v1.GenesisState
```
I don't know why that is.